docs/server/source/appendices/run-with-docker.md

```diff
@@ -78,6 +89,16 @@ docker run \
   --restart=always \
   --volume $HOME/bigchaindb_docker:/data \
   rethinkdb:2.3
+
+# For Docker for Mac users
```

Comment:
can you please add the docker version pre requisit since newer version dont use docker-machine while older needed that

Reply:
The older version with docker-machine not called `Docker for Mac`. `Docker for Mac` is specifically introduced to run docker directly on MacOS not having a proxy(default docker-machine.)